src/main/java/seedu/address/model/person/Remark.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+/**
+ * Represents a Person's remark in the address book.
+ */
+public class Remark {
+
+    public static final String MESSAGE_REMARK_CONSTRAINTS =
+            "The remark of a person can take any value.";
+
+    public final String value;
```

Comment:
Is `public` the best access modifier for `value`? As a rule of thumb start all data members as private, and depending on their usage promote them to protected, and if absolutely necessary, to public.